api/src/api/package.rs

```diff
@@ -229,8 +229,14 @@ pub async fn list_handler(
   let db = req.data::<Database>().unwrap();
   db.get_scope(&scope).await?.ok_or(ApiError::ScopeNotFound)?;
 
-  let (total, packages) =
-    db.list_packages_by_scope(&scope, start, limit).await?;
+  let iam = req.iam();
+  let (total, packages) = if iam.check_scope_admin_access(&scope).await.is_ok()
```

Comment:
Maybe or if the user is admin (with sudo enabled?). May not be needed if we add a package admin page.

Reply:
`check_scope_admin_access` already does that